src/Style.js

```diff
@@ -250,10 +255,12 @@ export const createSetStyleTag = (instance) => {
 
     const doNotPrefixFinal = instance.getCoalescedPropsValue(doNotPrefix, doNotPrefixGlobal);
     const isMinifiedFinal = instance.getCoalescedPropsValue(isMinified, isMinifiedGlobal);
+    // not using memoized version for autoPrefixerOpts as memoizing object will be more expensive than undefined check
```

Comment:
its a valid point ... if you wanted to remove the memoized application of `getCoalescedPropsValue` on all the props then I would not mind.